### PR TITLE
feat: pass other div props to Modal Container

### PR DESCRIPTION
**Pull request by ganiirsyadi**

### CODE REVIEW

File: src/@next/Modal/Modal.tsx [hunk 1]
```diff
@@ -53,6 +53,7 @@ export const Modal = React.forwardRef<HTMLDivElement, ModalProps>(
       closeOnClickOutside,
       onClose,
       onBack,
+      ...props
```
Comment (marthayaputra) [inline src/@next/Modal/Modal.tsx:56]: Don't you need to extend the ModalProps component with DivElement? Or did forwardRef took care of it?

Comment (ganiirsyadi) [reply src/@next/Modal/Modal.tsx:56]: I don't think so, i will add it

Comment (ganiirsyadi) [reply src/@next/Modal/Modal.tsx:56]: Added

File: src/@next/Modal/Modal.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ import {
   StyledModalBackButton,
 } from './ModalStyle';
 
-export type ModalProps = {
+export interface ModalProps extends React.HTMLAttributes<HTMLDivElement> {
```
Comment (ganiirsyadi) [inline src/@next/Modal/Modal.tsx:19]: Added

**Review by marthayaputra — APPROVED** [review]

**Pull request merged**